src/app/App.js

```diff
@@ -35,53 +46,56 @@ function App() {
   };
 
   useEffect(() => {
-    // const getUsers = async () => {
-
-    //   const data = await getDocs(usersCollectionRef);
-    //   console.log(data)
-    //   setUsers(data.docs.map((doc) => ({ ...doc.data(), id: doc.id })));
-    // };
-
-    // getUsers();
     dispatch(fetchTodos("123"));
   }, []);
 
+  useEffect(() => {
+    dispatch(fetchUsers("jackson"));
+  }, []);
+
   return (
-    <div className="App">
-      <input
-        placeholder="Name..."
-        onChange={(event) => {
-          setNewName(event.target.value);
-        }}
-      />
-      <input
-        type="number"
-        placeholder="Age..."
-        onChange={(event) => {
-          setNewAge(event.target.value);
-        }}
-      />
-      <button onClick={createUser}>Create User</button>
+    <div className={styles.App}>
+      <div className={styles.createUser}>
+        <Input
+          placeholder="Name..."
+          onChange={(event) => {
+            users.name(event.target.value);
+          }}
+        />
+        <Input
+          type="number"
+          placeholder="Age..."
+          onChange={(event) => {
+            users.age(event.target.value);
+          }}
+        />
+        <Button type="primary" onClick={createUser}>
+          Create User
+        </Button>
+      </div>
       {users.map((user) => {
         return (
           <div>
             {" "}
             <h1>NAME: {user.name}</h1>
             <h1>AGE: {user.age}</h1>
-            <button
+            <Button
+              type="primary"
               onClick={() => {
                 updateUser(user.id, user.age);
```

Comment:
Update should take in a new value, like from local state, that the user can modify to make updates to the user instead of updating with existing values